packages/client/src/components/Modals/SearchPanel.vue

```diff
@@ -11,12 +11,85 @@
         right
         shadow
       >
-      <b-form-group class="form">
-        <div class="multiselect-group">
-          <div class="multiselect-title">Category</div>
-          <multiselect v-model="opportunityCategoryFilters" :options="opportunityCategoryOptions" :multiple="true" :limit="1" :limitText="customLimitText" :close-on-select="false" :clear-on-select="false" placeholder="Opportunity Category" :show-labels="false" :searchable="false"></multiselect>
-        </div>
-      </b-form-group>
+        <form ref="form" class="search-form">
+          <b-form-group label-for="include-input">
+            <template slot="label">Include Keywords</template>
+              <b-form-input
+                id="include-input"
+                type="text"
+                v-model="formData.includeInput"
+              ></b-form-input>
+          </b-form-group>
+          <b-form-group label-for="exclude-input">
+            <template slot="label">Exclude Keywords</template>
+              <b-form-input
+                id="exclude-input"
+                type="text"
+                v-model="formData.excludeInput"
+              ></b-form-input>
+          </b-form-group>
+          <b-form-group label-for="oportunity-number">
```

Comment:
Typo "oportunity -> opportunity" lines 31, 32, 34, 36, 39, 41, 42, 44, 124, 125

Reply:
ah, good catch. thank you!

Reply:
should be fixed.